src/Dotnet.Script.DependencyModel/Runtime/RuntimeDependencyResolver.cs

```diff
@@ -67,21 +66,32 @@ private IEnumerable<RuntimeDependency> GetDependenciesInternal(string pathToProj
             List<string> nuGetPackageFolders = dependencyInfo.NugetPackageFolders.ToList();
             nuGetPackageFolders.Add(RuntimeHelper.GetPathToNuGetStoreFolder());
 
-            var runtimeDepedencies = new HashSet<RuntimeDependency>();
+            var runtimeDepedencies = new List<RuntimeDependency>();
```

Comment:
typo

Reply:
Should think I could spell "dependencies" by now 😄 Fixing